test/content-security-policy.test.ts

```diff
@@ -581,4 +582,16 @@ describe("getDefaultDirectives", () => {
       contentSecurityPolicy.getDefaultDirectives,
     );
   });
+
+  it("returns a new copy each time", () => {
+    const one = getDefaultDirectives();
+    one["worker-src"] = ["ignored.example"];
+    (one["img-src"] as Array<ContentSecurityPolicyDirectiveValue>).push(
```

Comment:
Rather than exporting `ContentSecurityPolicyDirectiveValue`, could we cast this to something like `Array<unknown>`, `Array<string>`, or even `any`?

Reply:
I did it just to be sure. My new commit removes the type export and uses `Array<string>`